client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/channel/EventTransport.java

```diff
@@ -64,7 +60,14 @@
      */
     void setEventManager(EventManager manager);
 
+    /**
```

Comment:
Please, get rid off these empty comment blocks or add some doc if it is useful there.